### ISSUE TITLE
Link to accessibility violation in the DOM

### ISSUE THREAD

**Issue by steven1046** [IBMa/equal-access#288]
Use this template if you want to request a new feature, or a change to an
existing feature.

If you are reporting a bug or problem, please use the bug template instead.

### Summary

I would like the IBM Equal Accessibility Checker tool (for Chrome) to link to the Elements tab showing where in the DOM the violation is.

### Justification

Very helpful and speeds up violation fixing.

### "Must have" functionality

Highlight any "must have" needs and functionality for the request.

**Comment by tombrunet** [comment]
Can you clarify? This should already happen if you are using the subpanel of the Elements tab. Or are using the "Accessibility Assessment" main panel? Chrome does not allow us to automatically switch to our subpanel. This is a known restriction in Chrome.

**Comment by steven1046** [comment]
Ah okay, i was talking about the accessibility assessment. Thanks, i'll close this if Chrome doesn't allow extensions to change tabs.

**Comment by tombrunet** [comment]
The intent was to provide some sort of link from the Element location to flip panels. Chrome requires a user to open a panel and does not let us do that programatically. We are keeping our eyes open for a work around. I think we may be able to inspect the element, but it would default to the CSS subpanels, which is also confusing.
